src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,135 @@
+package duke.tasks;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+import duke.tasks.*;
```

Comment:
Just curious if all of the tasks are required to be imported!

anyway, very neat stuff!! Good job! 👍🏼